README.md

```diff
@@ -1,2 +1,137 @@
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Budget <a name="budget"></a>
+
+**Budget** is a budgeting app build with rails, this is the capstone project for the Rails module, part of Microverse.org Fullstack program
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- **[Ruby](https://www.ruby-lang.org/en/)**
+- **[Ruby on Rails](https://rubyonrails.org/)**
+- **[JavaScript](https://www.javascript.com/)**
+- **[PostgreSQL](https://www.postgresql.org/)**
+- **[Bootstrap](https://getbootstrap.com/)**
+- **[RSpec](https://rspec.info/)**
+- **[Capybara](https://github.com/teamcapybara/capybara)**
+
+### Key Features <a name="key-features"></a>
+
+- **[Users can sign up and log in]
+- **[Users can create categories]
+- **[Users can create transactions]
+- **[Users can see, edit and delete their transactions and categories]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install [Ruby](https://www.ruby-lang.org/en/)
+- Install [Ruby on Rails](https://rubyonrails.org/)
+- Install [PostgreSQL](https://www.postgresql.org/)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+`git clone git@github.com:wickathou/budget.git`
+
+### Install
+
+Install the required gems
+`bundle install`
+
+Create the database by running the command
+`rails db:create`
+
+### Run tests
+
+Run the tests using
+`rspec`
+
+### Usage
+
+Then run the rails server
+
+`rails s`
+
+And now you should be able to see the project running on `localhost:3000`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Javier Hernandez**
+
+- GitHub: [@wickathou](https://github.com/wickathou)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/javierjhm/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Create clasification for expenses and income]**
+- [ ] **[Create a dashboard with charts]**
+- [ ] **[Create a budget for each clasification]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Leave a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- [Microverse](https://www.microverse.org/)
+
```

Comment:
- [ ] Kindly add the acknowledgment of the design which is necessary and also asked in your project requirements.
- [ ] The Original design idea is by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella).

- [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, Kindly update it in the README